### DISCUSSION TITLE
Missing code in direct-award-justification.gc

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#993] [Q&A] [answered]
One of our users is missing a value in the direct-award-justification.gc codelist: 

"The contract we want to conclude does not fall within the scope of Directive 2014/24/EU in accordance with Article 12(4) of that Directive. In the old Annex D1 of the standard form under point 2, you could also tick "The contract does not fall within the scope of this Directive". We have not found such an option here."

Are there plans to include this code in the codelist?

**Comment by YvesJo** [comment]
Hi,
This is more a question for DG GROW. I think this is something they are dealing with and invite you to get in touch with them directly to know more about its status/evolution.

**Reply by YvesJo** [reply, marked as answer]
There will be an additional code "non-dir" (or "non-directive") with SDK 1.13
